app/assets/javascripts/components/articles/add_available_articles.jsx

```diff
@@ -33,6 +36,18 @@ const AddAvailableArticles = createReactClass({
     this.props.open();
   },
 
+  async handleClick() {
```

Comment:
Duplicate code. Should be written out only once if possible.

Reply:
okay ty for pointing this out